src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTabRelatedArticles.java

```diff
@@ -154,6 +159,11 @@ public void propertyChange(PropertyChangeEvent evt) {
     }
 
 
+    /**
+     * Helper Class to initiate SwingWorker
+     *
```

Comment:
Remove empty lines

Can the class be private static?

See https://www.amazon.de/Effective-Java-2nd-Programming-Language/dp/0321356683?tag=jabref-21 for some background on this